me.mitya57.ReText.json

```diff
@@ -0,0 +1,378 @@
+                    ]
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "sed -r -i 's/^(\\[LIBXML_REQUIRED_VERSION=)/#\\1/' 'configure.ac';",
+                        "sed -r -i 's/^(\\[LIBXSLT_REQUIRED_VERSION=)/#\\1/' 'configure.ac';",
+                        "grep -Ei '_REQUIRED_VERSION' 'configure.ac' || :;"
+                    ]
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "autoreconf -i;"
+                    ]
+                }
+            ],
+            "post-install": [
+                "echo 'post-install: symlink';",
+                "[[ -f '/app/opt/xmlstarlet/bin/xmlstarlet' ]] || ln -s 'xml' '/app/opt/xmlstarlet/bin/xmlstarlet';"
+            ],
+            "cleanup": [
+                "/opt/xmlstarlet/share/doc",
+                "/opt/xmlstarlet/share/man"
+            ]
+        },
+        {
+            "name": "imagemagick",
+            "config-opts": [
+                "--prefix=/app/opt/imagemagick",
+                "--exec-prefix=/app/opt/imagemagick",
+                "--with-rsvg",
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.imagemagick.org/download/ImageMagick-6.9.9-42.tar.xz",
+                    "sha256": "873654cd55792cdaa9c3ee35f63e389fedfb3d34685a1f77d6b81e5c578ae527"
+                }
+            ],
+            "cleanup": [
+                "/opt/imagemagick/share/doc",
+                "/opt/imagemagick/share/man"
+            ]
+        },
+        {
+            "name": "python3-qt5",
+            "no-autogen": true,
+            "modules": [
+                {
+                    "name": "python3-sip",
+                    "no-autogen": true,
+                    "sources": [
+                        {
+                            "type":"file",
+                            "url":"https://pypi.python.org/packages/6b/46/2869369c8ec634ec9dc714b84834e9264acbe87eb7492328efccfa0c855e/sip-4.19.8-cp35-cp35m-manylinux1_x86_64.whl",
+                            "sha256":"e72955e12f4fccf27aa421be383453d697b8a44bde2cc26b08d876fd492d0174"
+                        },
+                        {
+                            "type": "shell",
+                            "commands": [
+                                "echo 'all:\n\ninstall:\n\tpip3 install --prefix=/app sip-*.whl;\n' > makefile;"
+                            ]
+                        }
+                    ]
+                }
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://pypi.python.org/packages/e4/15/4e2e49f64884edbab6f833c6fd3add24d7938f2429aec1f2883e645d4d8f/PyQt5-5.10.1-5.10.1-cp35.cp36.cp37.cp38-abi3-manylinux1_x86_64.whl",
```

Comment:
Build this from source.

Reply:
Updated:
https://github.com/scx/flathub/commit/b7594045334868fa38c133a0ad83801148c882d7#diff-8259601d3ba825e42e3782ae49e24c14